.github/workflows/changeset.yml

```diff
@@ -141,48 +134,3 @@ jobs:
         run: |
           echo "Please include at least one tag in the core changeset file"
           exit 1
```

Comment:
exit 1

Reply:
I think we still want to make sure that the changeset, if it exists, has tags.